pages/settings/index.html

```diff
@@ -1,139 +1,142 @@
+          </select>
         </div>
 
-        <div class="account-settings_content">
-          <section id="basicInfoSettings" style="display: none">
-            <h2 class="text-heading-2">Basic Info</h2>
+        <div class="account-settings__side-menu" id="sideMenu">
+          <a href="#" class="account-settings__menu-btn text-lg-body-1" id="basicInfoBtn">Basic Info</a>
+          <a href="#" class="account-settings__menu-btn text-lg-body-1" id="passwordBtn">Password</a>
+          <a href="#" class="account-settings__menu-btn text-lg-body-1" id="notificationsBtn">Notifications</a>
+          <a href="#" class="account-settings__menu-btn text-lg-body-1" id="securityBtn">Security</a>
+        </div>
+      </div>
 
-            <div class="account-settings__profile-picture">
-              <img src="../../assets/img/default-nav-image.png" alt="User profile picture" id="profilePhoto" />
+      <div class="account-settings_content">
+        <section id="basicInfoSettings" style="display: none">
+          <h2 class="text-heading-2">Basic Info</h2>
 
-              <div class="account-settings__profile-picture-btns">
-                <label class="btn btn-secondary" id="uploadImageBtn">
-                  Upload Image
-                  <input type="file" accept="image/*" id="inputImage" style="display: none" />
-                </label>
-                <button class="btn btn-error" id="deleteImageBtn">Delete</button>
-              </div>
-            </div>
+          <div class="account-settings__profile-picture">
+            <img src="../../assets/img/default-nav-image.png" alt="User profile picture" id="profilePhoto" />
 
-            <form id="profileForm">
-              <div class="form-field">
-                <label for="name" class="text-body-2 medium"> Name </label>
-                <div class="form-field__input-container">
-                  <input type="text" id="name" required class="text-body-2 regular" />
-                </div>
-              </div>
-            </form>
+            <div class="account-settings__profile-picture-btns">
+              <label class="btn btn-secondary" id="uploadImageBtn">
+                Upload Image
+                <input type="file" accept="image/*" id="inputImage" style="display: none" />
+              </label>
+              <button class="btn btn-error" id="deleteImageBtn">Delete</button>
+            </div>
+          </div>
 
-            <div class="account-settings__email form-field">
-              <label for="email" class="text-body-2 medium"> Email address </label>
+          <form id="profileForm">
+            <div class="form-field">
+              <label for="name" class="text-body-2 medium"> Name </label>
               <div class="form-field__input-container">
-                <input type="email" id="email" class="text-body-2 regular" disabled />
+                <input type="text" id="name" required class="text-body-2 regular" />
```

Comment:
I'm not sure if these changes are just formatting or if I moved things around. I'll check this in the morning.